src/main/java/it/pagopa/pn/deliverypush/service/impl/WebhookServiceImpl.java

```diff
@@ -173,12 +181,22 @@ private Mono<Void> processEvent(StreamEntity stream,  String oldStatus, String n
         }
 
         String timelineEventCategory = timelineElementInternal.getCategory().getValue();
+
+        List<String> filteredValues = new ArrayList<>();
+        if (eventType == StreamCreationRequest.EventTypeEnum.TIMELINE) {
+            filteredValues.addAll(stream.getFilterValues().isEmpty()
```

Comment:
eh ma c'è ancora l'addAll :)
io farei direttamente filteredValues = stream.getFilterValues().isEmpty()?defaultCategories:stream.getFilterValues()

Reply:
è un trick per evitare di trasformare Set in List, è un addAll su un ArrayList vuoto, un minimo di libertà implementativa si può avere?